# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xine-lib/xine-lib-1.1.8-dxr3.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---
This patch is from Gentoo
diff -ur xine-lib-1.1.8.orig/src/libffmpeg/ffmpeg_encoder.c xine-lib-1.1.8/src/libffmpeg/ffmpeg_encoder.c
--- ./src/libffmpeg/ffmpeg_encoder.c	2007-08-19 02:36:50.000000000 +0300
+++ ./src/libffmpeg/ffmpeg_encoder.c	2007-09-04 15:04:41.000000000 +0300
@@ -76,7 +76,7 @@
   lavc_data_t* this;
   avcodec_init();
 
-  register_avcodec(&mpeg1video_encoder);  
+  avcodec_register_all();
   lprintf("lavc init , version %x\n", avcodec_version());
   this = xine_xmalloc(sizeof(lavc_data_t));
   if (!this) return 0;
